packages/vuetify/src/composables/stack.ts

```diff
@@ -27,22 +32,33 @@ export function useStack (isActive: Readonly<Ref<boolean>>, zIndex: Readonly<Ref
   useToggleScope(isActive, () => {
     const lastZIndex = globalStack.at(-1)?.[1]
     _zIndex.value = lastZIndex ? lastZIndex + 10 : +zIndex.value
-    globalStack.push([vm.uid, _zIndex.value])
+
+    if (createStackEntry.value) {
+      globalStack.push([vm.uid, _zIndex.value])
+    }
 
     parent?.activeChildren.add(vm.uid)
 
     onScopeDispose(() => {
-      const idx = globalStack.findIndex(v => v[0] === vm.uid)
-      globalStack.splice(idx, 1)
+      if (createStackEntry.value) {
+        const idx = globalStack.findIndex(v => v[0] === vm.uid)
+        globalStack.splice(idx, 1)
+      }
 
       parent?.activeChildren.delete(vm.uid)
     })
   })
 
   const globalTop = ref(true)
   watchEffect(() => {
-    const _isTop = globalStack.at(-1)?.[0] === vm.uid
-    setTimeout(() => globalTop.value = _isTop)
+    if (createStackEntry.value) {
+      const _isTop = globalStack.at(-1)?.[0] === vm.uid
+      setTimeout(() => globalTop.value = _isTop)
+
+      return
+    }
+
+    globalTop.value = true
```

Comment:
If we turn `disableGlobalStack` into a `boolean` (not a ref containing a `boolean`), this line will not be needed because `true` is already the default value of `globalTop`.

Reply:
Just thinking about it... if we disable global stack for a particular overlay, should `globalTop` and `localTop` be `true`?